packages/gatekeeper/src/helpers/services.ts

```diff
@@ -94,15 +99,14 @@ export async function fetchUser(
 }
 
 /** some docstring */
```

Comment:
Could we please write a proper docstring?

Reply:
done